docs/userguides/compile.md

```diff
@@ -149,3 +143,17 @@ owner = accounts.test_accounts[0]
 
 instance = container.deploy(sender=owner)
 ```
+
+## Output Extra
+
+Sometimes, there are extra output styles you may want.
+For example, to output minified ABI JSONs, use the following config:
+
+```yaml
+compile:
+  output_extra:
+     - ABI
```

Comment:
how does this work?

Reply:
I didn't want to put back the split contract types in the .build folder. The ones who wanted that just wanted smaller files for hosting. So they want minified ABIs. Every compile, it dumps the manifest's ABIs to a directory when you enable this; for hosting

Reply:
so there's a list of accepted keys documented?